src/tools/number.ts

```diff
@@ -20,14 +20,21 @@ export function formatNumber(
     return nullValue;
   }
   const adjustedMinFractionDigits = Math.max(
-    minimumFractionDigits ?? fractionDigits,
+    Math.min(
+      minimumFractionDigits ?? fractionDigits,
```

Comment:
maximumFractionDigits should be considered if both maximumFractionDigits and fractionDigits are present.